packages/perseus/src/widgets/graded-group-set.jsx

```diff
@@ -262,6 +268,8 @@ const styles = StyleSheet.create({
     indicatorContainer: {
         display: "flex",
         flexDirection: "row",
+        listStyle: "none",
+        margin: "inherit",
```

Comment:
Just as a learning opportunity for me, what's this do?

Reply:
It takes the value of the parent element. A simple way to override the `ul` style without explicitly prescribing one in case some other style is needed in the future.

Reply:
Now that you've called my attention to it, there may be a more appropriate keyword to use here. Will experiment this morning.

Reply:
Changed to just `"unset"` to prevent any weird behavior